server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/EmlRoutesTest.java

```diff
@@ -0,0 +1,128 @@
+package org.apache.james.webadmin.routes;
+
+import static io.restassured.RestAssured.with;
+import static org.hamcrest.CoreMatchers.is;
+
+import org.apache.james.mailbox.MailboxSession;
+import org.apache.james.mailbox.exception.MailboxException;
+import org.apache.james.mailbox.inmemory.InMemoryId;
+import org.apache.james.mailbox.inmemory.manager.InMemoryIntegrationResources;
+import org.apache.james.mailbox.model.MailboxId;
+import org.apache.james.mailbox.model.MailboxPath;
+import org.apache.james.mailbox.store.StoreMailboxManager;
+import org.apache.james.webadmin.WebAdminServer;
+import org.apache.james.webadmin.WebAdminUtils;
+import org.apache.james.webadmin.service.EmlService;
+import org.eclipse.jetty.http.HttpStatus;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import io.restassured.RestAssured;
+import io.restassured.filter.log.LogDetail;
+
+public class EmlRoutesTest {
```

Comment:
I don't see test inspecting the mailboxManager to see that the message was well created with the right content.